### PR TITLE
code sniffer configs

### PR DESCRIPTION
**Pull request by ifarzana**

### CODE REVIEW

File: Dockerfile.codesniffer [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Stage 1
```
Comment (dasgarner) [inline Dockerfile.codesniffer:1]: Do you think we could reuse the `Dockerfile.ci` container? It does install dev dependencies, so should have the sniffer available.

File: .github/workflows/test-suite.yaml [hunk 2]
```diff
@@ -39,4 +39,17 @@ jobs:
...
           docker exec cms-db mysql -ucms -pjenkins cms -e "INSERT INTO oauth_client_scopes (clientId, scopeId) VALUES ('MrGPc7e3IL1hA6w13l7Ru5giygxmNiafGNhFv89d', 'all') ON DUPLICATE KEY UPDATE scopeId = scopeId"
-          docker run --ipc=host --name cms-cypress --link=cms-web:web -v $(pwd)/cypress.json:/app/cypress.json -v $(pwd)/cypress:/app/cypress xibosignage/xibo-cms:cypress bash -c "CYPRESS_baseUrl=http://web /app/node_modules/.bin/cypress run --config screenshotsFolder=/app/results,video=false --reporter junit --reporter-options 'mochaFile=/app/results/results_cypress_[hash].xml,toConsole=true'"
\ No newline at end of file
+          docker run --ipc=host --name cms-cypress --link=cms-web:web -v $(pwd)/cypress.json:/app/cypress.json -v $(pwd)/cypress:/app/cypress xibosignage/xibo-cms:cypress bash -c "CYPRESS_baseUrl=http://web /app/node_modules/.bin/cypress run --config screenshotsFolder=/app/results,video=false --reporter junit --reporter-options 'mochaFile=/app/results/results_cypress_[hash].xml,toConsole=true'"      - name: Run Cypress
```
Comment (dasgarner) [inline .github/workflows/test-suite.yaml:42]: Something odd has happened to this line

File: .github/workflows/test-suite.yaml [hunk 3]
```diff
@@ -39,4 +39,17 @@ jobs:
...
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      - name: Run Code Sniffer
```
Comment (dasgarner) [inline .github/workflows/test-suite.yaml:47]: I wonder if we should run the code sniffer before the tests, so we fail fast if we're going to?

File: .github/workflows/test-suite.yaml [hunk 4]
```diff
@@ -39,4 +39,17 @@ jobs:
...
           docker exec cms-db mysql -ucms -pjenkins cms -e "INSERT INTO oauth_client_scopes (clientId, scopeId) VALUES ('MrGPc7e3IL1hA6w13l7Ru5giygxmNiafGNhFv89d', 'all') ON DUPLICATE KEY UPDATE scopeId = scopeId"
-          docker run --ipc=host --name cms-cypress --link=cms-web:web -v $(pwd)/cypress.json:/app/cypress.json -v $(pwd)/cypress:/app/cypress xibosignage/xibo-cms:cypress bash -c "CYPRESS_baseUrl=http://web /app/node_modules/.bin/cypress run --config screenshotsFolder=/app/results,video=false --reporter junit --reporter-options 'mochaFile=/app/results/results_cypress_[hash].xml,toConsole=true'"
\ No newline at end of file
+          docker run --ipc=host --name cms-cypress --link=cms-web:web -v $(pwd)/cypress.json:/app/cypress.json -v $(pwd)/cypress:/app/cypress xibosignage/xibo-cms:cypress bash -c "CYPRESS_baseUrl=http://web /app/node_modules/.bin/cypress run --config screenshotsFolder=/app/results,video=false --reporter junit --reporter-options 'mochaFile=/app/results/results_cypress_[hash].xml,toConsole=true'"      - name: Run Cypress
+      - name: Dump GitHub context
```
Comment (dasgarner) [inline .github/workflows/test-suite.yaml:43]: Seems strange?

**Review by dasgarner — COMMENTED** [review]
Just some comments as I happened across the PR :-)

**Comment by ifarzana** [comment]
This can be merged now Dan. It finally works.

**Review by dasgarner — APPROVED** [review]

**Pull request merged**